HorizonCore/src/main/java/com/hubspot/horizon/HttpConfig.java

```diff
@@ -139,6 +151,8 @@ public static class Builder {
     private RetryStrategy retryStrategy = RetryStrategy.DEFAULT;
     private ObjectMapper mapper = new ObjectMapper();
     private SSLConfig sslConfig = SSLConfig.standard();
+    private String socksProxyHost = "";
```

Comment:
We should make it optional and default to `Optional.empty()` instead of using `""` as a sentinel value.